site/docs/2.0/components/navbar-off-canvas.md

```diff
@@ -12,34 +12,39 @@ Adding the `.navbar-offcanvas` class to your `<nav>` will incorporate an off-can
 
 {% capture example %}
 <div class="d-lg-none d-flex justify-content-end">
-  <button data-toggle="offcanvas" type="button" data-target="#navbarOffcanvasDemo" aria-controls="navbarOffcanvasDemo" class="btn btn-red">
-    <span class="material-icons-sharp"> menu </span>
+  <button data-toggle="offcanvas" type="button" data-target="#navbarOffcanvasDemo" aria-controls="navbarOffcanvasDemo" class="btn btn-red btn-menu">
```

Comment:
Should we update on this page as well? [Arizona Header](https://github.com/az-digital/arizona-bootstrap/blob/main/site/docs/2.0/components/arizona-header.md)?

Reply:
Maybe in a separate PR? Would we just need to add the same offcanvas menu html?

Reply:
New ticket here: https://github.com/az-digital/arizona-bootstrap/issues/213